docs/guide/runtime-routing.md

```diff
@@ -124,6 +124,9 @@ With the above configuration, the `site/offline` action will be used to handle a
 The `catchAll` property should take an array whose first element specifies a route, and
 the rest of the elements (name-value pairs) specify the parameters to be [bound to the action](structure-controllers.md#action-parameters).
 
+> Tip: In practice this is used in production, not at stage of development.
```

Comment:
I don't like this wording. It does not describe explicitly why it is not recommended to use this property on dev.

How about `Info: Debug panel on development environment will not work when this property is enabled.`